Composer/plugins/samples/assets/shared/scripts/provisionComposer.js

```diff
@@ -175,6 +176,30 @@ const unpackObject = output => {
   return unpacked;
 };
 
+/**
+ * For more information about this api, please refer to this doc: https://docs.microsoft.com/en-us/rest/api/resources/Tenants/List
+ * @param {*} accessToken 
+ */
+const getTenantId = async (accessToken) => {
+  if (!accessToken) {
+    throw new Error('Error: Missing access token. Please provide a non-expired Azure access token. Tokens can be obtained by running az account get-access-token');
+  }
+  try {
+    const tenantUrl = `https://management.azure.com/tenants?api-version=2020-01-01`;
+    const options = {
+      headers: { Authorization: `Bearer ${accessToken}` },
+    };
+    const response = await rp.get(tenantUrl, options);
+    const jsonRes = JSON.parse(response);
+    if (jsonRes.value === undefined || (jsonRes.value && jsonRes.value.length === 0) || (jsonRes.value && jsonRes.value.length > 0 && jsonRes.value[0].tenantId === undefined)) {
+      throw new Error(`Get empty tenantId from tenants API`);
```

Comment:
Small text improvement suggestion:

```suggestion
      throw new Error(`No tenants found in the account.`);
```